src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.vb

```diff
@@ -270,6 +272,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
                 For Each interfaceImpl In interfaceImpls
                     Dim interfaceHandle As EntityHandle = moduleSymbol.Module.MetadataReader.GetInterfaceImplementation(interfaceImpl).Interface
                     Dim namedTypeSymbol As NamedTypeSymbol = TryCast(tokenDecoder.GetTypeOfToken(interfaceHandle), NamedTypeSymbol)
+                    namedTypeSymbol = DirectCast(TupleTypeDecoder.DecodeTupleTypesIfApplicable(namedTypeSymbol, interfaceHandle, moduleSymbol), NamedTypeSymbol)
```

Comment:
`namedTypeSymbol` is created with `TryCast` above but `DecodeTupleTypesIfApplicable` does not handle `Nothing`.